lib/systems/default.nix

```diff
@@ -55,6 +55,7 @@ rec {
       # Misc boolean options
       useAndroidPrebuilt = false;
       useiOSPrebuilt = false;
+      isRemarkable = false;
```

Comment:
```suggestion
```
This is not how this is done

Reply:
@jkopanski Suggested this to me. What's the alternative?

Reply:
If you take all of my suggestions, since this becomes much smaller, `isRemarkable` is no longer as needed.